app/assets/stylesheets/user.scss

```diff
@@ -0,0 +1,62 @@
+@import "application";
+
+.sign-up-container {
+  border: 2px;
+  font-family: "Open Sans", sans-serif;
+  position: relative;
+  top: 8em;
+  width: 30% !important;
+
+}
+
+.sign-up-header {
+  background-color: $my-brown;
+  border-radius: 8px 8px 0 0;
+  color: $my-white;
+  font-family: "Open Sans", sans-serif;
+  font-size: 1.0em;
+  font-weight: lighter;
+  height: 7em;
+  opacity: 0.95;
+  padding: 1em;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+}
+
+.sign-up-box {
+  background-color: rgba(56,65,69,0.4);
```

Comment:
Color literals like `rgba(56, 65, 69, 0.4)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space